x/incentive/keeper/keeper_test.go

```diff
@@ -306,7 +307,7 @@ func (suite *KeeperTestSuite) TestIterateUSDXMintingAccrualTimes() {
 		return false
 	})
 
-	suite.Equal(expectedAccrualTimes, actualAccrualTimes)
+	suite.ElementsMatch(expectedAccrualTimes, actualAccrualTimes)
```

Comment:
The order for these change, hence the change to `ElementsMatch`, does this matter in this case?

Reply:
The order doesn't matter for any accrual times. It's set by the store, items are sorted by their keys.

Reply:
Got it, just fixed the `nonEmptyAccrualTimes` test accrual times order, updating the non empty denom to "ukava" changes the order.